CI Failure - Javadoc issues

> Error reading file: /usr/local/lib/android/sdk/docs/reference/element-list

```
javadoc: error - Error reading file: /usr/local/lib/android/sdk/docs/reference/element-list
> Task :rsdroid:androidJavadocs
1 error


> Task :rsdroid:androidJavadocs FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.1.1/userguide/command_line_interface.html#sec:command_line_warnings
18 actionable tasks: 5 executed, 13 up-to-date
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':rsdroid:androidJavadocs'.
> Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): '/home/runner/work/Anki-Android-Backend/Anki-Android-Backend/rsdroid/build/tmp/androidJavadocs/javadoc.options'

```

--- a/.github/workflows/linux_build.yml
+++ b/.github/workflows/linux_build.yml
@@ -55,3 +55,7 @@ jobs:
 
     - name: Build
       run: ./gradlew clean assembleRelease -DtestBuildType=release -Dorg.gradle.daemon=false -Dorg.gradle.console=plain # assembleAndroidTest
+
+    # Our publish workflow (publish_library.yaml) in on Ubuntu and needs javadocs (#57)
+    - name: Test Javadoc
+      run: ./gradlew :rsdroid:androidJavadocs -DtestBuildType=release -Dorg.gradle.daemon=false -Dorg.gradle.console=plain

